nservicebus/hosting/index.md

```diff
@@ -100,6 +107,10 @@ Related:
  * [Injecting the Bus into ASP.NET MVC Controller](/samples/web/asp-mvc-injecting-bus/)
 
 
+NOTO: Since V6 `IEndpointInstance` (the equivalent of `IBus`) is no longer automatically injected into the container. Here's a sample code showing how to do it using the Autofac container
```

Comment:
I think that statement is not true. Shouldn't we advise to bind directly the factory? So that they can inject `IBusContext`?